packages/api/src/schema/api-schema.yaml

```diff
@@ -686,6 +686,12 @@ components:
             example:
               - id not provided
               - user not found
+    multistream-target-input:
+      oneOf:
+        - $ref: "#/multistream-target"
```

Comment:
I don't think this ref will work. I simplified it on the message I wrote.

Reply:
lol, I just copy pasted it without actually look the string content! fixing it